.azure_pipelines/olive-ort-nightly.yaml

```diff
@@ -0,0 +1,84 @@
+trigger: none
+
+pr: none
+
+schedules:
+- cron: '0 0 * * *'
+  displayName: Daily build
+  branches:
+    include:
+    - main
+
+variables:
+    ComponentDetection.Timeout: 2400
+
+jobs:
+- template: job_templates/olive-build-template.yaml
+  parameters:
+    name: Linux_CI
+    pool: $(OLIVE_POOL_UBUNTU2004)
+    windows: False
+    onnxruntime: ort-nightly
+    examples:
+      bert_ptq_cpu:
+        exampleFolder: bert
+        exampleName: bert_ptq_cpu
+      bert_ptq_cpu_aml:
+        exampleFolder: bert
+        exampleName: bert_ptq_cpu_aml
```

Comment:
there is no use running the aml and docker tests since they are still using stable onnxruntime.

Reply:
same for multiple ep test too

Reply:
good cache